src/main/java/dao/basket/BasketDaoJDBC.java

```diff
@@ -0,0 +1,128 @@
+package dao.basket;
+
+import factory.UserServiceFactory;
+import model.Basket;
+import model.ConfirmationCode;
+import model.Product;
+import model.User;
+import org.apache.log4j.Logger;
+import service.user.UserService;
+import utils.DBConnector;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class BasketDaoJDBC implements BasketDao {
+
+    private static final UserService userService = UserServiceFactory.getInstance();
+    private static final Logger logger = Logger.getLogger(BasketDaoJDBC.class);
+    private static final String ADD_PRODUCT_TO_BASKET = "INSERT INTO list_of_products_in_basket " +
```

Comment:
try to use more understandable name of table. for exmpl: basket_product

Reply:
fixed